test/test_docstring_detection.py

```diff
@@ -0,0 +1,44 @@
+import tokenize
+from unittest import TestCase
+
+from flake8_quotes import Token, get_docstring_tokens
+from test.test_checks import get_absolute_path
+
+
+class GetDocstringTokensTests(TestCase):
+    def _get_docstring_tokens(self, filename):
+        with open(get_absolute_path(filename), 'r') as f:
+            tokens = [Token(t) for t in tokenize.generate_tokens(f.readline)]
+        return get_docstring_tokens(tokens)
+
+    def test_get_docstring_tokens_absent(self):
```

Comment:
I can separate the files for test_checks and test_docstring_detection, as in the first case we just need one docstring to test quote handling and for the second we need all the cases. I'm a bit skeptical about separating the module, class, function docstring tests, as while they might work in isolation, there could be bugs when all of them are present, which we would miss.

I don't know why you're focusing on indentation. Indentation doesn't matter regarding something being or not being a docstring. Our code ignores indentation. Basically nothing in flake8-quotes's code works around indentation. Why would indentation matter and why would we design tests around something that we really don't care about?